deskpro/deskpro-admin-frontend
# Fix Avatar with Text cutting off characters

Issue:
![Screen Shot 2020-01-24 at 6 17 38 PM](https://user-images.githubusercontent.com/46844842/73089122-dd4d2c80-3ed5-11ea-9bd0-1bf390feed81.png)

Test steps:

- Go to /#/tickets/departments
- Observe "hotdogs" line.
- See other rows for more examples

Expected behavior:
![Screen Shot 2020-01-24 at 6 17 19 PM](https://user-images.githubusercontent.com/46844842/73089127-e211e080-3ed5-11ea-8b7e-12928b470ade.png)
